docs/content/basic/installing_for_developers.rst

```diff
@@ -38,23 +37,26 @@ Working with Git and GitHub
 
 
 To keep track of your code changes and contribute back to SimPEG, you will
-need a github_ account and fork the `SimPEG repository <http://github.com/simpeg/simpeg>`_
+need a github_ account, then fork the `SimPEG repository <http://github.com/simpeg/simpeg>`_
+to your local account.
 (`How to fork a repo <https://docs.github.com/en/get-started/quickstart/fork-a-repo>`_). Software
```

Comment:
Remove "Software" from this sentence.
```suggestion
(`How to fork a repo <https://docs.github.com/en/get-started/quickstart/fork-a-repo>`_).
```

Reply:
I wish black could format `rst` files for line-length.